java/dagger/internal/codegen/Binding.java

```diff
@@ -49,6 +42,16 @@
 import javax.lang.model.type.DeclaredType;
 import javax.lang.model.type.TypeMirror;
 import javax.lang.model.util.SimpleTypeVisitor6;
+import kotlinx.metadata.Flag;
+import kotlinx.metadata.KmClass;
+
+import static com.google.common.base.Suppliers.memoize;
```

Comment:
Revert :)

Reply:
Yeah for some reason the GJF IDE plugin doesn't quite match what the command line/CI check expects. Would be good if the repo shipped with an autoformatter of some sort

Reply:
The GJF IDE plugin doesn't handle imports, this has to be configured manually.